### PR TITLE
Api Diff between preview 8 and preview 7

### PR DESCRIPTION
**Pull request by Anipik**
cc @tannergooding @GrabYourPitchforks @pgovind @eiriktsarpalis @layomia @tommcdon @krwq @dotnet/ncl @steveharter @bartonjs @Jozkee 

@Tratcher @Pilchie @ericstj

### CODE REVIEW

**Review by ManickaP — APPROVED** [review]
`System.Net.Http` and `System.Net.Http.Json` LGTM

**Comment by bartonjs** [comment]
System.Security.Cryptography and System.Security.Cryptography.X509Certificates LGTM

File: release-notes/5.0/preview/api-diff/preview8/.Net/5.0-preview8_System.Numerics.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# System.Numerics
```
Comment (tannergooding) [inline release-notes/5.0/preview/api-diff/preview8/.Net/5.0-preview8_System.Numerics.md:1]: System.Numerics LGTM

CC. @pgovind

File: release-notes/5.0/preview/api-diff/preview8/.Net/5.0-preview8_System.Runtime.Intrinsics.Arm.md [hunk 2]
```diff
@@ -0,0 +1,350 @@
+# System.Runtime.Intrinsics.Arm
```
Comment (tannergooding) [inline release-notes/5.0/preview/api-diff/preview8/.Net/5.0-preview8_System.Runtime.Intrinsics.Arm.md:1]: ARM Intrinsics LGTM.

CC. @CarolEidt, @echesakovMSFT, @kunalspathak

File: release-notes/5.0/preview/api-diff/preview8/.Net/5.0-preview8_System.Runtime.Intrinsics.X86.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# System.Runtime.Intrinsics.X86
```
Comment (tannergooding) [inline release-notes/5.0/preview/api-diff/preview8/.Net/5.0-preview8_System.Runtime.Intrinsics.X86.md:1]: X86 Intrinsics LGTM.

CC. @CarolEidt, @echesakovMSFT, @kunalspathak

File: release-notes/5.0/preview/api-diff/preview8/.Net/5.0-preview8_System.Buffers.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# System.Buffers
```
Comment (tannergooding) [inline release-notes/5.0/preview/api-diff/preview8/.Net/5.0-preview8_System.Buffers.md:1]: System.Buffers LGTM

CC. @GrabYourPitchforks, @pgovind

**Comment by Pilchie** [comment]
Tagging @pranavkm for the ASP.NET changes.

File: release-notes/5.0/preview/api-diff/preview8/.Net/5.0-preview8_System.Numerics.md (hunk 1 above)
Comment (pgovind) [reply release-notes/5.0/preview/api-diff/preview8/.Net/5.0-preview8_System.Numerics.md:1]: Numerics LGTM!

File: release-notes/5.0/preview/api-diff/preview8/.Net/5.0-preview8_System.Buffers.md (hunk 4 above)
Comment (pgovind) [reply release-notes/5.0/preview/api-diff/preview8/.Net/5.0-preview8_System.Buffers.md:1]: System.Buffers also looks good!

**Comment by layomia** [comment]
System.Text.Json and System.Collections LGTM

**Comment by Tratcher** [comment]
:shipit: AspNetCore looks good, but you'll still want @pranavkm to look at MVC and Components.

File: release-notes/5.0/preview/api-diff/preview8/Asp.Net/5.0-preview8_Microsoft.AspNetCore.Mvc.ApplicationModels.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Microsoft.AspNetCore.Mvc.ApplicationModels
+
+``` diff
+ namespace Microsoft.AspNetCore.Mvc.ApplicationModels {
+     public class PageParameterModel : ParameterModelBase, IBindingModel, ICommonModel, IPropertyModel {
+-        IReadOnlyList<object> Microsoft.AspNetCore.Mvc.ApplicationModels.ICommonModel.Attributes { get; }
```
Comment (pranavkm) [inline release-notes/5.0/preview/api-diff/preview8/Asp.Net/5.0-preview8_Microsoft.AspNetCore.Mvc.ApplicationModels.md:6]: This looks strange. `ICommonModel.Attributes` hasn't been removed.

File: release-notes/5.0/preview/api-diff/preview8/Asp.Net/5.0-preview8_Microsoft.AspNetCore.Authentication.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+# Microsoft.AspNetCore.Authentication
+
+``` diff
+ namespace Microsoft.AspNetCore.Authentication {
+     public class AuthenticationBuilder {
+-        public virtual AuthenticationBuilder AddPolicyScheme<TService>(string authenticationScheme, string displayName, Action<PolicySchemeOptions, TService> configureOptions) where TService : class;
```
[the remark this answers is not in the document]
Comment (HaoK) [reply release-notes/5.0/preview/api-diff/preview8/Asp.Net/5.0-preview8_Microsoft.AspNetCore.Authentication.md:6]: This is expected, we reverted adding this overload

**Pull request merged**